index.js

```diff
@@ -31,6 +32,22 @@ const argv = require('yargs')
         desc: 'Print an HTML file or URL to PDF',
         builder: {
             ...commonOptions,
+            'emulatemedia': {
+                string: true,
```

Comment:
configure this as a boolean

Reply:
It's not a boolean, it's "screen" "print" etc.